skywalking/log/sw_logging.py

```diff
@@ -75,6 +77,9 @@ def build_log_tags() -> LogTags:
 
         context = get_context()
 
+        if config.agent_log_print_tid:
+            record.tid = str(context.segment.related_traces[0])
```

Comment:
We add the `tid` info to the `record` only when `agent_log_print_tid == true`, but what if I set `SW_AGENT_LOG_REPORTER_LAYOUT='%(tid)s %(asctime)s [%(threadName)s] %(levelname)s %(name)s - %(message)s` but `agent_log_print_tid == false`? The implementation looks unreasonable to me, we have two ways to modify the log layout which might be conflict with each other.

Reply:
@kezhenxu94 thanks for your comment, my opinion is:
1) If and only if agent_log_print_tid=true, the agent will enable the ability to print traceId.
2) When the user-defined layout has tid, but agent_log_print_tid=false, printing traceId is turned off
3) Or emphasize in the document that when agent_log_print_tid=true, the layout will change accordingly.

Reply:
How about we make it more consistent with the Java agent, that is simply introducing the tid mechnism through the custom layout?

Reply:
> How about we make it more consistent with the Java agent, that is simply introducing the tid mechnism through the custom layout?

This makes sense to me